app/models/cmr.rb

```diff
@@ -355,24 +355,45 @@ def self.get_raw_granule(concept_id)
     granule_results["Granule"]
   end
 
+  # Given a granule concept id, it will pull all revisions and return the latest granule revision.
   def self.get_raw_granule_results(concept_id)
```

Comment:
If this does in fact return a "single" granule, I think it should be renamed to "get_raw_granule"